libs/custodian/regions.go

```diff
@@ -112,37 +112,37 @@ type GeoAllowBlockMap struct {
 }
 
 func (gabm GeoAllowBlockMap) Verdict(countries ...string) bool {
-
 	var (
 		allow bool
 		block bool
 	)
-
-OUTER_ALLOW:
-	for _, ac := range gabm.Allow {
-		for _, country := range countries {
-			if strings.EqualFold(ac, country) {
-				// in allow list
-				allow = true
-				break OUTER_ALLOW
+	if len(gabm.Allow) > 0 {
```

Comment:
It seems like this function is doing two things either allow or block, its a bit confusing. If it is to be one function could this not go into a contains function, something like

```
// ...
func Verdict {
	if len(gabm.Allow) > 0 {
		return contains(countries, gabm.Allow)
	}	
	return contains(gabm.Block, countries)
}

func contains(countries, allowblock []string) bool {
	for _, ab := range allowblock {
		for _, country := range countries {
			if strings.EqualFold(ab, country) {
				return true
			}
		}
	}
	return false
}
```
then call that in the Allow or Block sections.

Reply:
good idea, implemented here: https://github.com/brave-intl/bat-go/pull/1647/commits/3514d418d691747b7c876910047a20d90b1a7bfa